src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/suggestions-settings-dialog.component.spec.ts

```diff
@@ -228,10 +225,8 @@ class TestEnvironment {
     tick();
   }
 
-  changeSelectValue(element: HTMLElement, option: number): void {
-    const selectElem = element.querySelector('select');
-    selectElem!.value = option.toString();
-    selectElem!.dispatchEvent(new Event('change'));
+  changeSelectValue(mdcSelect: MdcSelect, option: number): void {
+    mdcSelect.setSelectionByValue(option.toString());
```

Comment:
Done

Reply:
Indeed. But as discussed, the old way is no longer possible because it no longer uses a native select.

You did mention though that you think elsewhere in the tests we click the menu, find the menu item we're looking for, and click it. I searched our codebase and found only two other instances like this, and in both we used this method. So for now we should probably leave it as-is, but an improvement could be made. Or perhaps there is a place where we do it differently and you may be able to locate it.